### ISSUE TITLE
[bug] CMakeToolchain produces broken toolchains

### ISSUE THREAD

**Issue by ohanar** [conan-io/conan#7830]
If more than one variable or definition is set, the CMakeToolchain produces broken toolchain files. They are missing necessary newlines. Functionality was broken by https://github.com/conan-io/conan/commit/a77b97817a73fbdd2fbeb5f7038ad400aafd1803.

### Environment Details (include every applicable attribute
  * Conan version: 1.30
  * Python version: 3.6, 3.7, 3.8

### Steps to reproduce (Include if Applicable)

Use a cmake package with the following toolchain method:

```python
class MyClass(ConanFile):
    ...
    def toolchain(self):
        toolchain = CMakeToolchain(self)
        toolchain.variables["MY_FIRST_CMAKE_VARIABLE"] = "foo"
        toolchain.variables["MY_SECOND_CMAKE_VARIABLE"] = "bar"
        toolchain.write_toolchain_files()
```

**Comment by memsharded** [comment]
Submitted fix for next patch 1.30.1 in https://github.com/conan-io/conan/pull/7833

**Comment by memsharded** [comment]
Closed by https://github.com/conan-io/conan/pull/7833, will be in 1.30.1
